.gitignore

```diff
@@ -159,9 +159,15 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 .idea/
 
+# Visual Studio Code
+.vscode/
+
 # Misago Sandbox Demo Repo
 src/sandbox/misago-demo/Misago
 
+# Django-machina whoosh search index
```

Comment:
I would add a comment basically saying what you did in the summary that files are generated by the indexer and put here based on the settings. 

Please link to relevant documentation, expand on why this is the specific directory, and why it should be in the gitignore